scripts/lint.py

```diff
@@ -67,6 +68,9 @@ def lint_line(line: str, file_extension: str = "rs", is_in_docstring: bool = Fal
     if double_the.search(line.lower()):
         return "Found 'the the'"
```

Comment:
we obviously no longer need the the the filter now :)

Reply:
It it slightly broader, ignoring case. I couldn't make the general one as broad, as it lead to too many false positives.
But the narrow double-words catches _most_ of the double-the:s